test/unit/ModuleTest.php

```diff
@@ -4,12 +4,12 @@
 
 namespace Boesing\Psr\Http\Message\Multipart;
 
+use Boesing\Psr\Http\Message\Multipart\Module;
```

Comment:
shouldn't be necessary as the namespace is already `Boesing\Psr\Http\Message\Multipart` but I guess that has been added by codesniffer? 🤷🏼‍♂️